### PR TITLE
Feat/add label right to atom switch single

### PR DESCRIPTION
**Pull request by fredyzz**
### Types of changes

- [x] New feature (non-breaking change which adds functionality)

### Description, Motivation and Context

Add right label to single type switch

### Screenshots - Animations
Captura de pantalla 2021-08-24 a las 13 05 32

### CODE REVIEW

**Review by andresin87 — APPROVED** [review]

**Comment by andresin87** [comment]
❤️

**Review by Alverd04 — APPROVED** [review]
👌

File: components/atom/switch/demo/index.js [hunk 1]
```diff
@@ -111,23 +111,33 @@ const TypesArticle = () => (
       This package gives 3 different <Code>type</Code> values provided under the{' '}
```
Comment (turolopezsanabria) [inline components/atom/switch/demo/index.js:111]: We should update this line as well

File: components/atom/switch/demo/index.js [hunk 2]
```diff
@@ -111,23 +111,33 @@ const TypesArticle = () => (
...
+      <Cell key="typeEmptyCell" style={flexCenteredStyle}>
+        <Label>Single right</Label>
+      </Cell>
       {Object.values(atomSwitchTypes).map((type, index) => (
         <Cell key={index} style={flexCenteredStyle}>
           <AtomSwitch
             labelLeft="labelLeft"
-            labelRight="labelRight"
+            labelRight={type !== atomSwitchTypes.SINGLE ? 'labelRight' : ''}
             label={<Small>label</Small>}
```
Comment (turolopezsanabria) [inline components/atom/switch/demo/index.js:128]: we don't need to include these documentation small labels in the demo

**Review by nistadev — APPROVED** [review]
👌🏽

File: components/atom/switch/demo/index.js (hunk 1 above)
Comment (andresin87) [reply components/atom/switch/demo/index.js:111]: there are only 3 different types, but 'single' one have different behaviors

Comment (fredyzz) [reply components/atom/switch/demo/index.js:111]: We have improved the demo, to show clearly the 3 types, and the two behaviors of the SINGLE type.

Captura de pantalla 2021-08-25 a las 8 37 43

File: components/atom/switch/demo/index.js (hunk 2 above)
Comment (fredyzz) [reply components/atom/switch/demo/index.js:128]: Done

**Review by oriolpuig — APPROVED** [review]

File: components/atom/switch/demo/index.js (hunk 1 above)
Comment (turolopezsanabria) [reply components/atom/switch/demo/index.js:111]: great!

**Review by turolopezsanabria — APPROVED** [review]

**Pull request merged**